### ISSUE TITLE
Reddit URLs not supported

### ISSUE THREAD

**Issue by george-haddad** [postlight/mercury-parser#220]
The following reddit url will not get parsed correctly

[https://www.reddit.com/r/BetterEveryLoop/comments/9yz5o2/warmth/](https://www.reddit.com/r/BetterEveryLoop/comments/9yz5o2/warmth/)

```bash
curl -i -X GET \
   -H "Content-Type:application/json" \
   -H "x-api-key:xxxxxxxx" \
 'https://mercury.postlight.com/parser?url=https%3A%2F%2Fwww.reddit.com%2Fr%2FBetterEveryLoop%2Fcomments%2F9yz5o2%2Fwarmth%2F'
```

Returns

```bash
HTTP/1.1 502 Bad Gateway
Content-Length: 36
Content-Type: application/json
Date: Tue, 22 Jan 2019 09:19:41 GMT
Via: 1.1 501ad2910f631f0520a6d389d6f053e9.cloudfront.net (CloudFront)
X-Amz-Apigw-Id: T5f1iE-nIAMF8cg=
X-Amz-Cf-Id: HIkrESehbkfLFequ3G2yuVC0J2jf5nsGQeo2b1t_7_1IPV4B3JDuzA==
X-Amzn-Requestid: d3573917-1e26-11e9-8168-4922f69c2a07
X-Cache: Error from cloudfront
Age: 10
Connection: keep-alive
Server: Netlify
X-NF-Request-ID: 3006aa4c-a2b7-46c8-b9cf-57fb160bc529-581598

{"message": "Internal server error"}
```

Before and after screenshots using mercury reader

![merc_before](https://user-images.githubusercontent.com/3069650/51523807-1ff8a680-1e35-11e9-9d92-28c351e01e7c.png)

![merc_after](https://user-images.githubusercontent.com/3069650/51523810-21c26a00-1e35-11e9-948e-1d0ffaf0fcf4.png)

**Comment by ryanmcgrath** [comment]
Curious - is this still an issue if you sub in `old.reddit.com`? It could be a redesign-centric issue, no?

**Comment by jamespo** [comment]
old.reddit.com URLs work

**Comment by toufic-m** [comment]
This has been addressed in #307 and will be included in the next release.
